File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auto/AutoBlueTop.java
package org.firstinspires.ftc.teamcode.auto;

import com.qualcomm.robotcore.eventloop.opmode.Autonomous;

import org.firstinspires.ftc.teamcode.enums.Alliance;
import org.firstinspires.ftc.teamcode.enums.Location;

/**
 * Holds callable AutoBlueTop OpMode.
 */


@SuppressWarnings("unused")
@Autonomous(name = "Auto Blue Top", group = "auto")
public class AutoBlueTop extends AbstractAuto {
    public AutoBlueTop() {
        super(Alliance.BLUE, Location.UPPER);
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auto/AutoBlueTop.kt
package org.firstinspires.ftc.teamcode.auto

import com.qualcomm.robotcore.eventloop.opmode.Autonomous
import org.firstinspires.ftc.teamcode.enums.Alliance
import org.firstinspires.ftc.teamcode.enums.Location

/**
 * Holds callable AutoBlueTop OpMode.
 */


@Autonomous(name = "Auto Blue Right", group = "auto")
class AutoBlueTop : Auto(Alliance.BLUE, Location.UPPER)
